### PR TITLE
[WIP] DustAppRunAction support

### PR DESCRIPTION
**Pull request by spolu**
- Adds the capability to run the "latest" specification from `core`
- Adds inputs generation based on Dust app input dataset schema and description
- Adds the execution of the Dust app as DustAppRunAction
- Plugs new events all the way to the UI but no UI yet.
- Adds model rendering of DustAppRun outputs

Deployment:
- deploy `core`
- run `init_db` on `front`
- Deploy `front`

### CODE REVIEW

File: front/components/assistant/conversation/AgentAction.tsx [hunk 1]
```diff
@@ -11,5 +12,10 @@ export function AgentAction({ action }: { action: AgentActionType }) {
       </div>
     );
   }
+  if (isDustAppRunActionType(action)) {
+    <div className="pb-4">
+      <span className="text-red">TODO(spolu) DUST APP RUN ACTION</span>
+    </div>;
```
Comment (lasryaric) [inline front/components/assistant/conversation/AgentAction.tsx:18]: I assume this is intended

File: front/lib/api/assistant/actions/dust_app_run.ts [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+      "Error generating DustAppRun action inputs"
+    );
+
+    return new Err(rawInputsRes.error);
+  }
+
+  return new Ok({});
+}
+
+// Event sent during before the execution of a dust app run with the finalized params to be used.
```
Comment (lasryaric) [inline front/lib/api/assistant/actions/dust_app_run.ts:179]:
```suggestion
// Event sent before the execution of a dust app run with the finalized params to be used.
```

File: front/lib/api/assistant/agent.ts [hunk 3]
```diff
@@ -158,7 +165,10 @@ export type AgentErrorEvent = {
 };
 
 // Event sent durint the execution of an action. These are action specific.
```
Comment (lasryaric) [inline front/lib/api/assistant/agent.ts:167]:
```suggestion
// Event sent during the execution of an action. These are action specific.
```

File: front/lib/models/assistant/actions/dust_app_run.ts [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+      defaultValue: DataTypes.NOW,
+    },
+    updatedAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+    sId: {
+      type: DataTypes.STRING,
+      allowNull: false,
```
Comment (lasryaric) [inline front/lib/models/assistant/actions/dust_app_run.ts:48]:
```suggestion
      allowNull: false,
      unique:true,
```

**Review by lasryaric — CHANGES_REQUESTED** [review]
Cool! 🔥 Made a few small suggestions.

File: front/components/assistant/conversation/AgentAction.tsx (hunk 1 above)
Comment (spolu) [reply front/components/assistant/conversation/AgentAction.tsx:18]: removed the ; but yeah intended for now :+1:

**Review by lasryaric — DISMISSED** [review]

**Pull request merged**